### ISSUE TITLE
Auto-generated linker script for EFR32xG24 Pro Kit looks incorrect

### ISSUE THREAD

**Issue by dsyx** [SiliconLabs/gecko_sdk#42]
The environment I use is as follows:

- IDE: Simplicity Studio, v5.6.3.1
- SDK: Gecko SDK, v4.2.2
- Board: EFR32xG24 Pro Kit +20 dBm

I created a Bluetooth application based on the example template **Bluetooth - SoC Blinky** without making any changes to the template. After flashing this application image to the board, I found that the application was not working properly. After a bit of poking around, I found out that there is a problem with `autogen/linkerfile.ld`, its original content is:

```
 MEMORY
 {
   FLASH   (rx)  : ORIGIN = 0x8012000, LENGTH = 0x16c000
   RAM     (rwx) : ORIGIN = 0x20000000, LENGTH = 0x40000
 }
```
I modified it to:

```
 MEMORY
 {
   FLASH   (rx)  : ORIGIN = 0x8000000, LENGTH = 0x16c000
   RAM     (rwx) : ORIGIN = 0x20000000, LENGTH = 0x40000
 }
```

After rebuilding and flashing to the board, the application works fine.

**Comment by silabs-PhilippL** [comment]
That's not an actual issue. You just did not flash the bootloader. The bootloader is supposed to be located in the flash memory area 0x8000000 - 0x8012000.
If you don't want to use a bootloader, just uninstall the "Bootloader Application Interface" component from your project's software components and the linkerfile should adopt accordingly using flash memory base address 0x8000000 then.

**Comment by dsyx** [comment]
@silabs-PhilippL 
You are right. However most projects depend on a lot of SOFTWARE COMPONENTS, is there documentation for these components and when they should be installed or uninstalled?
